aerosol_concentrations/README.md

```diff
@@ -23,9 +23,8 @@ middle of the month as opposed to its last day.
 The output is a NetCDF file that contains concentrations (in kg/kg, dry mass)
 for a set of aerosols defined on a lon-lat-z-time grid.
 
-The script also produces a `lowres` version. The `lowres` version contains fewer
-aerosol, one year of data only, it is at 1/6th of the resolution, and
-single-point precision.
+The script also produces a `lowres` version. The `lowres` version contains two
+years of data, it is at 1/6th of the resolution, and single-point precision.
```

Comment:
It might be nice to explain that it contains the first and last year of the input data specifically

Reply:
Added